x/superfluid/README.md

```diff
@@ -695,9 +716,12 @@ superfluid staking than just LP shares. Each AssetType has a different
 algorithm used to get its "Osmo equivalent value".
 
 We represent different types of superfluid assets as different enums.
-Currently, only enum `1` is actually used. Enum value `0` is reserved
-for the Native staking token for if we deprecate the legacy staking
-workflow to have native staking also go through the superfluid module.
+Currently, enum `1` represent s Gamm shares and enum `2` Concentrated 
```

Comment:
"represents s Gamm shares" I assume is a typo

Reply:
https://github.com/osmosis-labs/osmosis/pull/8331/commits/7a62c33feac4dfcb8fbe33d95634d71e70d6edf0